### DISCUSSION TITLE
Problem downloading DASH file with latest version of companion app

### DISCUSSION THREAD

**Discussion by Anoneemouse** [aclap-dev/video-downloadhelper#387] [Help and Q&A]
### Your question

Hi,

I downloaded and installed the latest version of the companion app today so I could download videos from ok.ru (when I tried with an older version of the comp. app., it said I had to update it).  I'm using Windows 10.

I downloaded one DASH file from that site with no problems at all (just using the VDH icon dropdown & clicking on the desired resolution/video size).  I have it set to default to the last folder used for the save location. A little bit later, I tried to download another, and the download arrow changed from a downward facing download arrow to the horizontal facing triangle for "open" - which I assumed meant it was either downloading or had downloaded the file.  But the toolbar icon isn't showing a download in progress, nor does it show up in my download history, and I can't find the video anywhere on my drive either.  

I wasn't sure if DASH downloads using the companion app required waiting 2 hours like HLS files do... and I probably didn't wait 2 hours - but it didn't go to the page saying I had to wait either like it does if you try to download a second HLS file before 2 hours have passed - so I assumed from all of these things that it was either downloading or had downloaded the file, and yet it's not on my drive anywhere that I can find. 

So I have no idea what's going on or why I can't seem to download this file.  Also, I thought once a file had been downloaded and the down arrow changed to the sideways triangle that there was still an option to download it again if one clicked on the detail arrow on the righthand side... but I don't see any option to try downloading it again. Here's the page I'm trying to download from:  https://ok.ru/video/5021293873870

Thanks in advance for help with this issue!

### Addon & CoApp version and details

version: 8.2.2.9
channel: stable
build date: 2024-03-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\noney",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by Mattmjs** [comment]
So it's acting like it was downloaded but you're sure you didn't. I couldn't replicate it.
Go into settings/more and reload extension
then reload the page which should fix it.

**Reply by Anoneemouse** [reply]
Thanks Mattmjs!

Actually, before I saw your reply but quite some time after I'd posted the problem, it had apparently been long enough that VDH was no longer "seeing" the video. So I reloaded the page again, hit download as I had before, and it downloaded just fine. So maybe it really was the 120 minute wait, only it wasn't showing me that was the issue?  Usually it'll open a new tab with a warning about having to wait 120 minutes between downloads, but when I had this problem, it hadn't done that, so I had no clue what was going on.

**Reply by Mattmjs** [reply]
There is no 120 minutes limit on Dash.
But there is on HLS video, something to keep in mind. Firefox doesn't have the this limitation.

**Reply by Anoneemouse** [reply]
Oh!  Ok, that's really good to know - and before your earlier post, I didn't know I could go into settings/more to reload the extension - so I really appreciate that info too!!  At least I know what to do now if this strange quirk happens again.  

I've really got to install the current version of firefox.  I have a really old version of it that I kept because of an app I use to update a small website online (really ought to switch to a current program for that too, but I'm familiar with the app in the old firefox, before it went quantum).  Anyhow, so long as I'm still using the old firefox, to install the new one also, I have to figure out again how to have two different versions of firefox with their own profiles and be sure that I don't screw up the old version of firefox and it's existing profile when I try to add the new firefox... sigh.  I'd done that once years ago on an older computer - and at first totally screwed up the old version of firefox, then had troubles finding it somewhere where I could download and install it again, find and reinstall all the apps I had on it, and so on.  I did finally get it set up right and working for both versions.  But it's been so long since I did that, I don't remember exactly how to do it!  :-(  I really should figure it out tho - I actually much prefer Firefox for regular browsing anyhow, yet have sat here using chrome for several years now.  

Anyhow, thanks so much for the info on VDH!!!
